src/renderer/services/binance/service.ts

```diff
@@ -280,9 +287,18 @@ const loadAssetTxs$ = ({
   const endTime = Date.now()
   // 90 day window - similar to ASGARDEX wallet approach,
   // see https://gitlab.com/thorchain/asgard-wallet/-/blob/develop/imports/api/wallet.js#L39-48
-  const diffTime = 90 * 24 * 60 * 60 * 1000
+  const diffTime = 90 * 24 * 60 * 60 * 1001
   const startTime = endTime - diffTime
-  return Rx.from(client.getTransactions({ txAsset, endTime, startTime, limit, offset })).pipe(
+  return Rx.from(
+    // @TODO add txAsset to the xchain
```

Comment:
^ Pls refer to a GH issue (or create one if we don't have it yet).

Reply:
@veado it was a tmp TODO for my self. this is already done at the xchain package just missed this TODO comment. will just remove it in few momemnt